src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -91,20 +88,13 @@ fs.readdirSync = jest.fn().mockReturnValue([])
   isDirectory: () => file_path === MOCK_FLARE_FOLDER_PATH || file_path === MOCK_CWD,
 }))
 
-// Zip mocks
-jest.mock('jszip')
-const mockJSZip = {
-  file: jest.fn(),
-  generateAsync: jest.fn().mockResolvedValue('zip content'),
-}
-;(JSZip as any).mockImplementation(() => mockJSZip)
-
```

Comment:
This mock is not needed anymore since we moved the zip logic and zip tests to helpers